src/components/SocialMedia.tsx

```diff
@@ -0,0 +1,39 @@
+import "../_styles/index.scss";
+
+import { FunctionComponent } from "react";
+import { Icon } from "./Icon";
+import Column from "./layout/Column";
+import Container from "./layout/Container";
+import Row from "./layout/Row";
+
+const SocialMedia: FunctionComponent = () => {
+  return (
+    <Container className="social-media">

```

Comment:
It make sense to not wrap social media component in <Container>, since it will be re-used in different places. Let's simplify structure and use the same approach like it is in menu:
```
<ul>
     <li>
       <a
            href="https://www.linkedin.com/in/anastasiia-sorina-b9681686"
            target="_blank"
          >
            <Icon type="linkedIn" />
          </a>
     </li>
     ....
</ul>
```
Where `ul` has styles `display:flex; flex-direction: column` and space between `li` managed by `margin-bottom:...`.

Reply:
- [ ] Done [501c15b](https://github.com/AnastasiSo/Portfolio-Project/pull/12/commits/501c15be0a2154e0f0aca6168a64c62f97c90f28)